crates/solidity/outputs/npm/package/generated/index.d.ts

```diff
@@ -623,12 +623,35 @@ export class RuleNode {
   get type(): NodeType.Rule;
   get kind(): RuleKind;
   get textLength(): TextIndex;
-  get children(): (RuleNode | TokenNode)[];
+  get children(): Array<RuleNode | TokenNode>;
 }
 export class TokenNode {
   get type(): NodeType.Token;
   get kind(): TokenKind;
   get textLength(): TextIndex;
+  get text(): string;
+}
+export class Cursor {
+  clone(): Cursor;
+  spawn(): Cursor;
+  get isFinished(): boolean;
+  get node(): RuleNode | TokenNode | null;
+  get textOffset(): TextIndex;
+  get textRange(): TextRange;
+  get pathRuleNodes(): Array<RuleNode>;
+  goToNext(): boolean;
+  goToNextNonDescendent(): boolean;
+  goToPrevious(): boolean;
+  goToParent(): boolean;
+  goToFirstChild(): boolean;
+  goToLastChild(): boolean;
+  goToNthChild(childNumber: number): boolean;
+  goToNextSibling(): boolean;
+  goToPreviousSibling(): boolean;
+  findRuleKind(kinds: Array<RuleKind>): RuleNode | null;
```

Comment:
TOL: thoughts about unifying `findXXXX()` and `findXXXXWithoutRecursing()` to a default argument `findXXXX(recursive = true)`?

Reply:
I've removed the without recursing forms, but in any case I wouldn't want to add a recursive param to findXXX if the use cases were not equaly distributed, which in this case I suspect they wouldn't be.